test_conformance/math_brute_force/function_list.cpp

```diff
@@ -278,7 +313,7 @@ const Func functionList[] = {
     ENTRY(minmag, 0.0f, 0.0f, FTZ_OFF, binaryF),
     ENTRY(modf, 0.0f, 0.0f, FTZ_OFF, unaryF_two_results),
     ENTRY(nan, 0.0f, 0.0f, FTZ_OFF, unaryF_u),
-    ENTRY(nextafter, 0.0f, 0.0f, FTZ_OFF, binaryF),
+    ENTRY(nextafter, 0.0f, 0.0f, FTZ_OFF, binaryF_nextafter),
```

Comment:
I think all the `ENTRY` items here should be updated to use the correct ULPs as defined by the spec - https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_Ext.html#cl_khr_fp16-relative-error-as-ulps

This is the same concerns as when I reviewed the previous PR https://github.com/KhronosGroup/OpenCL-CTS/pull/529#discussion_r368026314

Reply:
I've corrected ULP values for half test cases. I have two additional remarks which popped up in the process:

1) There are couple of tests focused on kernel procedures already covered by `test_relationals`, there are also other kernel procedures which should land in `test_relationals` for consistency, list below:

 ```
    isequal
    isfinite
    isgreater
    isgreaterequa
    isinf
    isless
    islessequal
    islessgreater
    isnan
    isnormal
    isnotequal
    isordered
    isunordered
```

Should we raise a new issue for that task ?

2) It looks like current progress of `test_bruteforce` (new PR [#1681](https://github.com/KhronosGroup/OpenCL-CTS/pull/1681)) does not take into account difference in processing for embedded profile for `cl_khr_fp16`. Should it be covered by new issue or continuation of development for PR #1681 ?

Reply:
I would suggest documenting both these points as their own issues to keep the scope of changes in #1681 manageable

Reply:
Done, I've created issues #1684 and #1685. Since we have already PR #1681 should we close this PR to avoid future confusion ?

Reply:
@gwawiork Are you okay if we close this PR in favour of #1681 so that reviwers don't review the wrong PR accidentally? Closing won't delete the git branch, and we can always reopen the PR later for whatever reason if we had to.

Reply:
@EwanC can we assume silence means consent ?

Reply:
Sure, let's close this. We can always reopen it if closing the PR was an issue for whatever reason.